### DISCUSSION TITLE
Should the Height and Width sliders be flipped?

### DISCUSSION THREAD

**Discussion by JoshuaKimsey** [hlky/stable-diffusion#156] [General]
_Wanted to do this as a discussion in case it's not actually a bug._

This is a curious question, and it may well be a regional thing between America/The West and other parts of the world. But, I sorta just noticed that it feels a bit odd having Height listed first, followed by Width. Typically in the US, and in Europe I think to, we list Width first, followed by Height, such as 1920x1080. I've definitely messed up a couple of generations thinking I was adjusting width when it was height, and vice-versa.

If this is simply a regional thing between the US and the world, then that's cool, it probably shouldn't be messed with. Otherwise, maybe those two should be flipped to fit with the typical naming convention for Width and Height? \

Curious what @hlky @altryne and other's thoughts are on this as well. Maybe I just discovered a unique regional quirk I never knew about. :)

**Comment by JoshuaKimsey** [comment]
Seems #158 might also be related to this too. I hadn't actually noticed this discrepancy.

**Comment by altryne** [comment]
100%, prob.  an oversite, should be consistent with the way it's done in dreamstudio as well. 
I'll add replace in my working branch

**Reply by hlky** [reply]
I fixed this on dev forgot to accept pull request on main

**Reply by JoshuaKimsey** [reply]
Awesome!
